app/controllers/users_controller.rb

```diff
@@ -0,0 +1,28 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
+    return if @user
+
+    redirect_to root_path
+  end
```

Comment:
```suggestion
  def show; end
```
E đưa code bên trong hàm show ra thành hàm load_user, call trong before_action nha